From f6477b487741738528bbc74741ef00885fdea56c Mon Sep 17 00:00:00 2001
From: maximilian attems <max@stro.at>
Date: Tue, 14 Jun 2011 22:54:35 +0200
Subject: [PATCH] Fix build for patch(1) users

fixes:
make: execvp: /usr/src/linux-3.0-rc3/scripts/depmod.sh: Permission denied
Signed-off-by: maximilian attems <max@stro.at>
---
 Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile b/Makefile
index badb923..b6230c4 100644
--- a/Makefile
+++ b/Makefile
@@ -1526,7 +1526,7 @@ quiet_cmd_rmfiles = $(if $(wildcard $(rm-files)),CLEAN   $(wildcard $(rm-files))
 
 # Run depmod only if we have System.map and depmod is executable
 quiet_cmd_depmod = DEPMOD  $(KERNELRELEASE)
-      cmd_depmod = $(srctree)/scripts/depmod.sh $(DEPMOD) $(KERNELRELEASE)
+      cmd_depmod = $(CONFIG_SHELL) $(srctree)/scripts/depmod.sh $(DEPMOD) $(KERNELRELEASE)
 
 # Create temporary dir for module support files
 # clean it up only when building all modules
-- 
1.7.2.5

